internal/contentcache/contentcache.go

```diff
@@ -44,15 +48,39 @@ func (c *ContentCache) RecoverCache() error {
 	if c.tempDir == "" {
 		c.tempDir = "/tmp"
 	}
+	fmt.Printf("Recovering cache:\n")
 	files, err := ioutil.ReadDir(c.tempDir)
 	if err != nil {
 		// if we fail to read the specified directory, log and return error
 		return fmt.Errorf("recover cache: %w", err)
 	}
-	for _, file := range files {
+	for _, metadataFile := range files {
 		// validate not a directory and matches gcsfuse pattern
-		if !file.IsDir() && matchPattern(file.Name()) {
-			// TODO ezl: load the files from disk to the in memory map
+		if !metadataFile.IsDir() && matchPattern(metadataFile.Name()) {
+			var metadata gcsx.TempFileObjectMetadata
+			// TODO ezl inefficient string operation
+			metadataAbsolutePath := c.tempDir + "/" + metadataFile.Name()
+			contents, err := ioutil.ReadFile(metadataAbsolutePath)
+			// TODO ezl should we exec some sort of cleanup in the cache if there are errors
+			if err != nil {
+				fmt.Printf("skip metadata file %v due to read error: %s", metadataFile.Name(), err)
```

Comment:
This is a DEBUG message. Use `logger.NewDebug` to create a logger at DEBUG level and use that to write the debug logs. Logs should start with capitalized letters.